exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -42,266 +41,316 @@ git submodule add https://github.com/526avijitgupta/gokarna.git themes/gokarna
 
 ### c. Basic Configuration {#basic-configuration}
 
-The following is a basic configuration for the gokarna theme:
+A simple Hugo [configuration file](https://gohugo.io/getting-started/configuration/#configuration-file) with [menu items](https://gohugo.io/content-management/menus/#properties-front-matter).
+
+Gokarna supports [adding Feather icons to the header](/posts/theme-documentation-advanced/#icons-in-header).
 
 ```toml
-baseURL = "http://example.org/"
+baseURL = "https://example.org/"
 defaultContentLanguage = "en"
-languageCode = "en"
-
-title = "My New Hugo Site"
-
-theme = "gokarna"
-
 # Automatically generate robots.txt
 enableRobotsTXT = true
-
-# Automatically set last modified dates using Git commit dates
-enableGitInfo = true
```

Comment:
I'm not sure if we should retain the `enableGitInfo` property in the basic documentation. On one hand, it's a useful feature. On the other hand it adds some "noise" to the basic documentation and maybe it's better suited to the advanced documentation.
@yashmehrotra thoughts?

Reply:
This is something I wanted to defer to Advanced, so I agree

Reply:
+1 for advanced